### ISSUE TITLE
Support referencing console by build number

### ISSUE THREAD

**Issue by thisgeek** [cliffano/nestor#44]
Add an option to the `console` command which takes a build number value:

```bash
nestor console jobName --build 84
```

The command outputs the console for the specified build, rather than the latest one as the `console` command would do by default. I expect most people would use it to print out the console for completed builds.

**Comment by cliffano** [comment]
@thisgeek Thanks for the feature request.
To keep things consistent with the other commands, the console command format would look like this:
`nestor console <job_name> [build_number]`
with job_name being mandatory and build_number optional.

**Comment by cliffano** [comment]
@thisgeek I've released Nestor v1.0.2 which includes this feature.

`nestor console <job_name> [build_number]`

and as usual if no build number is supplied, then it will show the latest build

`nestor console <job_name>`

Closing this issue for now, please feel free to re-open / open new issue if there's any prob.

**Comment by thisgeek** [comment]
From the commit history it looks like v1.0.3 includes this feature.

**Comment by cliffano** [comment]
@thisgeek That version bump is actually for the next release, i.e. release v1.0.2 (which includes this console feature), and then bump up the version to v1.0.3 for the next release.

I tend to bump up version after a release to make sure that when people check master branch, the version in master is always the next one.
In the past I used to add -pre postfix to distinguish released version or not, but this practice isn't consistent across the noderatis, so I switched back to just using plain numbers.
In Java land, this is similar to having a -SNAPSHOT.
